sql/comments/public.sql

```diff
@@ -79,13 +123,48 @@ COMMENT ON COLUMN public.circulation_check_ins.occurred_date_time IS 'Date and t
+COMMENT ON COLUMN public.circulation_scheduled_notices.notice_config__template_id IS 'ID (Generated UUID) of related template';
+COMMENT ON COLUMN public.circulation_scheduled_notices.notice_config__timing IS 'Timing represents when we need to send notice, before, at or after loan due date';
 COMMENT ON COLUMN public.circulation_scheduled_notices.recipient_user_id IS 'ID (generated UUID) of the user to whom this notice should be sent to';
 COMMENT ON COLUMN public.circulation_scheduled_notices.request_id IS 'ID (generated UUID) of related request for request notices';
 COMMENT ON COLUMN public.circulation_scheduled_notices.triggering_event IS 'Scheduled notice triggering event';
-COMMENT ON COLUMN public.circulation_scheduled_notices."data" IS 'JSON representation of the record';
+COMMENT ON COLUMN public.circulation_scheduled_notices.data IS 'JSON representation of the record';
+
+--public.user_users
+COMMENT ON COLUMN public.user_users.id IS 'ID (generated UUID) for the user';
+COMMENT ON COLUMN public.user_users.active IS 'A flag to determine if a user can log in, take out loans, etc.';
+COMMENT ON COLUMN public.user_users.barcode IS 'The unique library barcode for this user';
+COMMENT ON COLUMN public.user_users.created_date IS 'Deprecated';
+COMMENT ON COLUMN public.user_users.enrollment_date IS 'The date when the user joined the organization';
+COMMENT ON COLUMN public.user_users.expiration_date IS 'The date when the user becomes inactive';
+COMMENT ON COLUMN public.user_users.metadata__created_by_user_id IS 'ID (generated UUID) of the user who created the record (when available)';
+COMMENT ON COLUMN public.user_users.metadata__created_date IS 'Date and time when the record was created';
+COMMENT ON COLUMN public.user_users.metadata__updated_by_user_ID IS 'ID (generated UUID) of the user who last updated the record (when available)';
+COMMENT ON COLUMN public.user_users.metadata__updated_date IS 'Date and time when the record was last updated';
+COMMENT ON COLUMN public.user_users.patron_group IS 'ID (generated UUID) corresponding to the group the user belongs to';
+COMMENT ON COLUMN public.user_users.personal__date_of_birth IS 'The users birth date';
+COMMENT ON COLUMN public.user_users.personal__email IS 'The users email address';
+COMMENT ON COLUMN public.user_users.personal__first_name IS 'The users given name';
+COMMENT ON COLUMN public.user_users.personal__last_name IS 'The users surname';
+COMMENT ON COLUMN public.user_users.personal__middle_name IS 'The users middle name (if any)';
+COMMENT ON COLUMN public.user_users.personal__mobile_phone IS 'The users mobile phone number';
+COMMENT ON COLUMN public.user_users.personal__phone IS 'The users primary phone number';
+COMMENT ON COLUMN public.user_users.personal__preferred_contact_type_id IS 'ID of users preferred contact type like Email, Mail or Text Message';
+COMMENT ON COLUMN public.user_users.personal__preferred_first_name IS 'The users preferred name';
```

Comment:
Pretty insignificant but maybe these need to be `user's` instead of just `users`?

Other then that - looks great to me!

Reply:
@nassibnassar Are there processing problems when single quotes there?

Reply:
@stdombek I added some double single quotes in so I think it should be okay now.

Reply:
I think the escaping here looked correct to me.